internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -276,10 +276,6 @@ func (*Sequencer) Boot(r runtime.Runtime) []runtime.Phase {
 	).Append(
 		"startEverything",
 		StartAllServices,
-	).AppendWhen(
-		r.State().Platform().Mode() != runtime.ModeContainer && !r.Config().Machine().Kubelet().SkipNodeRegistration(),
```

Comment:
I didn't see `runtime.ModeContainer` handled elsewhere :thinking:

Reply:
I wasn't sure why it matters in container mode.

Reply:
not sure either, maybe kubelet handles it differently?

Reply:
nope, I don't think it cares

Reply:
looks like tests are flaky